transactions/WFLY-14869-microprofile-lra.adoc

```diff
@@ -0,0 +1,123 @@
+= Support for MicroProfile LRA
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          transaction, saga, lra, microprofile
+
+== Overview
+
+The goal of this RFE is providing the Galleon feature pack for WildFly which integrates the
```

Comment:
@xstefank I don't think this RFE is about providing a feature pack. It is about integrating the LRA functionality into the existing 'wildfly' feature pack, by providing two new extensions along with Galleon layers.

There are a lot of wording changes below that are needed to deal with removing the notion that there are new feature packs here.

BTW I think this is just about analysis doc wording. The https://github.com/wildfly/wildfly/pull/16547 PR is what I expect; it's about adding functionality to the existing 'wildfly' feature pack.